x-pack/plugins/timelines/public/components/t_grid/toolbar/bulk_actions/alert_status_bulk_actions.tsx

```diff
@@ -98,7 +98,9 @@ export const AlertStatusBulkActionsComponent = React.memo<StatefulAlertStatusBul
             case 'in-progress':
               title = i18n.IN_PROGRESS_ALERT_SUCCESS_TOAST(updated);
           }
-          addSuccess({ title });
+          if (title) {
```

Comment:
I think instead of wrapping this in a conditional, it would make more sense to just define a title for the 'acknowledged' status and add that to the switch statement here https://github.com/elastic/kibana/pull/108215/files#diff-be41c6b8395bf7f002e9788b9199c5820fd7ac6cb3b65c093fdb780049c07a21R91 you can then remove the if statement and fire a success toast for all 4 cases, and not 3 of 4 and failing silently on the 4th which would be sort of odd.

Reply:
The `if` statements I added are because the `let` doesn't initialize a value, so even with a 4th `case`, `title` will be able to be `undefined`. `addSuccess()` is actually ok with that (I'm not sure what happens if it's undefined?), but `addErrror` can only accept a string. This seemed like the most straight-forward way to solve that but I'm open to other ideas.